### PR TITLE
Add document for new source cloudPcCrossCloudGovernmentOrganizationMapping and related APIs

### PR DESCRIPTION
**Pull request by SuyliuMS**
Add document for new source cloudPcCrossCloudGovernmentOrganizationMapping and related APIs.

### CODE REVIEW

File: changelog/Microsoft.CloudManagedDesktop.json [hunk 1]
```diff
@@ -1,5 +1,183 @@
 {
     "changelog": [
+        {
+            "ChangeList": [
+                {
```
Comment (AshleyYangSZ) [inline changelog/Microsoft.CloudManagedDesktop.json:5]: Only include related changeList here.

**Review by AshleyYangSZ — CHANGES_REQUESTED** [review]

File: api-reference/beta/resources/cloudpccrosscloudgovernmentorganizationmapping.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
+---
+title: "cloudPcCrossCloudGovernmentOrganizationMapping resource type"
+description: "**TODO: Add Description**"
```
Comment (RuiHou105) [inline api-reference/beta/resources/cloudpccrosscloudgovernmentorganizationmapping.md:3]: Please update the description here.

File: api-reference/beta/resources/cloudpccrosscloudgovernmentorganizationmapping.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+# cloudPcCrossCloudGovernmentOrganizationMapping resource type
+
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Represents Cloud PC organization mapping between public cloud organization and US government cloud organization.
+For GCC (US Government Community Cloud) customers, the Azure Active Directory for tenant is in public cloud, but the Azure resources and Windows 365 Cloud PCs are in US government cloud. This mapping is required for customer administrators to setup & config Windows 365 and for the end users at GCC customers to access their Windows 365 Cloud PCs. The setup and maintenance of the tenant mapping must be done while maintaining the security and compliance requirements for the FedRAMP certification and onboarding to the US Government cloud.
+
+Inherits from [entity](../resources/entity.md).
```
Comment (RuiHou105) [inline api-reference/beta/resources/cloudpccrosscloudgovernmentorganizationmapping.md:19]: I suppose we don't need this line.

File: api-reference/beta/api/cloudpccrosscloudgovernmentorganizationmapping-get.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+}
+-->
+``` http
+GET https://graph.microsoft.com/beta/deviceManagement/virtualEndpoint/crossCloudGovernmentOrganizationMapping
+X-MS-CloudPC-USGovCloudTenantAADToken: {token}
+```
+
+
+### Response
+>**Note:** The response object shown here might be shortened for readability.
```
Comment (RuiHou105) [inline api-reference/beta/api/cloudpccrosscloudgovernmentorganizationmapping-get.md:67]: This extra note can be removed.

File: api-reference/beta/api/cloudpccrosscloudgovernmentorganizationmapping-post.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+|Application|Not supported.|
+
+## HTTP request
+
+<!-- {
+  "blockType": "ignored"
+}
+-->
+``` http
+POST ** Collection URI for microsoft.graph.cloudPcCrossCloudGovernmentOrganizationMapping not found
```
Comment (RuiHou105) [inline api-reference/beta/api/cloudpccrosscloudgovernmentorganizationmapping-post.md:33]: POST /deviceManagement/virtualEndpoint/crossCloudGovernmentOrganizationMapping

File: api-reference/beta/api/cloudpccrosscloudgovernmentorganizationmapping-post.md [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+```
+
+## Request headers
+|Name|Description|
+|:---|:---|
+|Authorization|Bearer {token}. Required.|
+|Content-Type|application/json. Required.|
+|X-MS-CloudPC-USGovCloudTenantAADToken|{token}. Required.|
```
Comment (RuiHou105) [inline api-reference/beta/api/cloudpccrosscloudgovernmentorganizationmapping-post.md:41]: Please add more description for this header: Represents the AAD token of government cloud tenant.

File: api-reference/beta/api/cloudpccrosscloudgovernmentorganizationmapping-post.md [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+POST https://graph.microsoft.com/beta/deviceManagement/virtualEndpoint/cloudPcCrossCloudGovernmentOrganizationMapping
+Content-Type: application/json
+X-MS-CloudPC-USGovCloudTenantAADToken: {token}
+
+{}
+```
+
+
+### Response
+>**Note:** The response object shown here might be shortened for readability.
```
Comment (RuiHou105) [inline api-reference/beta/api/cloudpccrosscloudgovernmentorganizationmapping-post.md:68]: You can remove this extra line.

**Review by RuiHou105 — CHANGES_REQUESTED** [review]
Please resolve all comments in this PR. Thanks!

File: api-reference/beta/resources/cloudpccrosscloudgovernmentorganizationmapping.md [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+## Methods
+|Method|Return type|Description|
+|:---|:---|:---|
+|[Create cloudPcCrossCloudGovernmentOrganizationMapping](../api/cloudpccrosscloudgovernmentorganizationmapping-post.md)|[cloudPcCrossCloudGovernmentOrganizationMapping](../resources/cloudpccrosscloudgovernmentorganizationmapping.md)|Create a new [cloudPcCrossCloudGovernmentOrganizationMapping](../resources/cloudpccrosscloudgovernmentorganizationmapping.md) object.|
+|[Get cloudPcCrossCloudGovernmentOrganizationMapping](../api/cloudpccrosscloudgovernmentorganizationmapping-get.md)|[cloudPcCrossCloudGovernmentOrganizationMapping](../resources/cloudpccrosscloudgovernmentorganizationmapping.md)|Read the properties and relationships of a [cloudPcCrossCloudGovernmentOrganizationMapping](../resources/cloudpccrosscloudgovernmentorganizationmapping.md) object.|
+
+## Properties
+|Property|Type|Description|
+|:---|:---|:---|
+|id|String|The tenant ID of the GCC (US Government Community Cloud) tenant in public cloud. Inherited from [entity](../resources/entity.md).|
```
Comment (AshleyYangSZ) [inline api-reference/beta/resources/cloudpccrosscloudgovernmentorganizationmapping.md:28]: Could you help remove the `Inherited from [entity](../resources/entity.md).`?

**Review by AshleyYangSZ — CHANGES_REQUESTED** [review]

**Review by AshleyYangSZ — APPROVED** [review]

File: api-reference/beta/api/cloudpccrosscloudgovernmentorganizationmapping-post.md [hunk 9]
```diff
@@ -0,0 +1,86 @@
...
+## Examples
+
+### Request
+<!-- {
+  "blockType": "request",
+  "name": "create_cloudpccrosscloudgovernmentorganizationmapping_from_"
+}
+-->
+``` http
+POST https://graph.microsoft.com/beta/deviceManagement/virtualEndpoint/cloudPcCrossCloudGovernmentOrganizationMapping
```
Comment (RuiHou105) [inline api-reference/beta/api/cloudpccrosscloudgovernmentorganizationmapping-post.md:59]:
```suggestion
POST https://graph.microsoft.com/beta/deviceManagement/virtualEndpoint/crossCloudGovernmentOrganizationMapping
```

**Review by RuiHou105 — CHANGES_REQUESTED** [review]
@SuyliuMS 
1.One comment left. Please help resolve.
2. Please create a roadmap workitem and attach the number in comment. Here is an [example](https://microsoftgraph.visualstudio.com/Roadmap/_workitems/edit/15373/). Please do this firstly if no access: 
Request access via [Improved ADO Access - 1ES Wikirequest to join](https://www.1eswiki.com/wiki/Improved_ADO_Access) .
Join the Microsoft Graph Roadmap Users ([graphroadmap@microsoft.com](mailto:graphroadmap@microsoft.com)) security group on [idweb](https://idwebelements/GroupManagement.aspx?Group=graphroadmap&Operation=join) .

**Comment by SuyliuMS** [comment]
roadmap workitem number: 15387

**Review by RuiHou105 — APPROVED** [review]

**Comment by RuiHou105** [comment]
@Lauragra  @JarbasHorst please help review this PR when you got time. Thanks.

File: api-reference/beta/resources/cloudpccrosscloudgovernmentorganizationmapping.md [hunk 10]
```diff
@@ -0,0 +1,53 @@
+---
+title: "cloudPcCrossCloudGovernmentOrganizationMapping resource type"
+description: "Represents Cloud PC organization mapping between public cloud organization and US government cloud organization."
```
Comment (Danielabom) [inline api-reference/beta/resources/cloudpccrosscloudgovernmentorganizationmapping.md:3]:
```suggestion
description: "Represents a Cloud PC organization mapping between a public and US government cloud organizations."
```

File: api-reference/beta/resources/cloudpccrosscloudgovernmentorganizationmapping.md [hunk 11]
```diff
@@ -0,0 +1,53 @@
...
+doc_type: resourcePageType
+---
+
+# cloudPcCrossCloudGovernmentOrganizationMapping resource type
+
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Represents Cloud PC organization mapping between public cloud organization and US government cloud organization.
```
Comment (Danielabom) [inline api-reference/beta/resources/cloudpccrosscloudgovernmentorganizationmapping.md:16]:
```suggestion
Represents a Cloud PC organization mapping between a public and US government cloud organizations.
```

File: api-reference/beta/resources/cloudpccrosscloudgovernmentorganizationmapping.md [hunk 12]
```diff
@@ -0,0 +1,53 @@
...
+---
+
+# cloudPcCrossCloudGovernmentOrganizationMapping resource type
+
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Represents a Cloud PC organization mapping between a public and US government cloud organizations.
+For GCC (US Government Community Cloud) customers, the Azure Active Directory for tenant is in public cloud, but the Azure resources and Windows 365 Cloud PCs are in US government cloud. This mapping is required for customer administrators to setup & config Windows 365 and for the end users at GCC customers to access their Windows 365 Cloud PCs. The setup and maintenance of the tenant mapping must be done while maintaining the security and compliance requirements for the FedRAMP certification and onboarding to the US Government cloud.
```
Comment (Danielabom) [inline api-reference/beta/resources/cloudpccrosscloudgovernmentorganizationmapping.md:17]:
```suggestion
For GCC (US Government Community Cloud) customers, the Azure Active Directory for the tenant is in a public cloud, but the Azure AD resources and Windows 365 Cloud PCs are in the US government cloud. Therefore, tenant mapping should be set up and maintained while updating the security and compliance requirements for the FedRAMP certification and onboarding to the US Government cloud. Tenant mapping is required for customer administrators when setting up & configuring Windows 365 and for GCC end users accessing their Windows 365 Cloud PCs.
```

File: api-reference/beta/resources/cloudpccrosscloudgovernmentorganizationmapping.md [hunk 13]
```diff
@@ -0,0 +1,53 @@
...
+|:---|:---|:---|
+|[Create cloudPcCrossCloudGovernmentOrganizationMapping](../api/cloudpccrosscloudgovernmentorganizationmapping-post.md)|[cloudPcCrossCloudGovernmentOrganizationMapping](../resources/cloudpccrosscloudgovernmentorganizationmapping.md)|Create a new [cloudPcCrossCloudGovernmentOrganizationMapping](../resources/cloudpccrosscloudgovernmentorganizationmapping.md) object.|
+|[Get cloudPcCrossCloudGovernmentOrganizationMapping](../api/cloudpccrosscloudgovernmentorganizationmapping-get.md)|[cloudPcCrossCloudGovernmentOrganizationMapping](../resources/cloudpccrosscloudgovernmentorganizationmapping.md)|Read the properties and relationships of a [cloudPcCrossCloudGovernmentOrganizationMapping](../resources/cloudpccrosscloudgovernmentorganizationmapping.md) object.|
+
+## Properties
+|Property|Type|Description|
+|:---|:---|:---|
+|id|String|The tenant ID of the GCC (US Government Community Cloud) tenant in public cloud.|
+|organizationIdsInUSGovCloud|String collection|The ID of the tenant in Azure Government corresponding to the GCC (US Government Community Cloud) tenant in public cloud. We only support 1:1 mapping for now, so this collection can only contains 1 tenant ID.|
```
Comment (Danielabom) [inline api-reference/beta/resources/cloudpccrosscloudgovernmentorganizationmapping.md:29]: Could we update the first sentence: Azure Government ____? 
My suggestion for the second sentence: We only support 1:1 mapping for now, so this collection can only contain one tenant ID.

File: api-reference/beta/api/cloudpccrosscloudgovernmentorganizationmapping-post.md [hunk 14]
```diff
@@ -0,0 +1,86 @@
+---
```
Comment (Danielabom) [inline api-reference/beta/api/cloudpccrosscloudgovernmentorganizationmapping-post.md:1]: Please rename this file based on the guidelines: https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211751/File-naming

File: changelog/Microsoft.CloudManagedDesktop.json [hunk 15]
```diff
@@ -1,5 +1,31 @@
...
+                    "Id": "ce998c71-ecc3-4fe7-adaa-ff6a2ced7cee",
+                    "ApiChange": "Resource",
+                    "ChangedApiName": "cloudPcCrossCloudGovernmentOrganizationMapping",
+                    "ChangeType": "Addition",
+                    "Description": "Added the [cloudPcCrossCloudGovernmentOrganizationMapping](https://docs.microsoft.com/en-us/graph/api/resources/intune-cloudPcCrossCloudGovernmentOrganizationMapping?view=graph-rest-beta) resource",
+                    "Target": "cloudPcCrossCloudGovernmentOrganizationMapping"
+                },
+                {
+                    "Id": "ce998c71-ecc3-4fe7-adaa-ff6a2ced7cee",
+                    "ApiChange": "Resource",
```
Comment (Danielabom) [inline changelog/Microsoft.CloudManagedDesktop.json:15]:
```suggestion
                    "ApiChange": "Relationship",
```

File: changelog/Microsoft.CloudManagedDesktop.json [hunk 16]
```diff
@@ -1,5 +1,31 @@
...
+                    "ChangedApiName": "crossCloudGovernmentOrganizationMapping",
+                    "ChangeType": "Addition",
+                    "Description": "Added the **crossCloudGovernmentOrganizationMapping** relationship to [virtualEndpoint](https://docs.microsoft.com/en-us/graph/api/resources/intune-virtualEndpoint?view=graph-rest-beta) resource.",
```
Comment (Danielabom) [inline changelog/Microsoft.CloudManagedDesktop.json:18]:
```suggestion
                    "Description": "Added the **crossCloudGovernmentOrganizationMapping** navigation property to [virtualEndpoint](https://docs.microsoft.com/en-us/graph/api/resources/intune-virtualEndpoint?view=graph-rest-beta) resource.",
```

**Review by Danielabom — CHANGES_REQUESTED** [review]
HI @SuyliuMS, thank you for working on this PR. Please review and address my comments. Thanks :)

File: api-reference/beta/resources/cloudpccrosscloudgovernmentorganizationmapping.md (hunk 13 above)
Comment (SuyliuMS) [reply api-reference/beta/resources/cloudpccrosscloudgovernmentorganizationmapping.md]: Fixed

File: api-reference/beta/api/cloudpccrosscloudgovernmentorganizationmapping-post.md (hunk 14 above)
Comment (SuyliuMS) [reply api-reference/beta/api/cloudpccrosscloudgovernmentorganizationmapping-post.md]: Fixed

File: api-reference/beta/resources/cloudpccrosscloudgovernmentorganizationmapping.md (hunk 13 above)
Comment (Danielabom) [inline api-reference/beta/resources/cloudpccrosscloudgovernmentorganizationmapping.md]:
```suggestion
|organizationIdsInUSGovCloud|String collection|The tenant ID in the Azure Government cloud corresponding to the GCC (US Government Community Cloud) tenant in the public cloud. Currently, we support 1:1 mappings, so this collection can only contain one tenant ID.|
```

Comment (SuyliuMS) [reply api-reference/beta/resources/cloudpccrosscloudgovernmentorganizationmapping.md]: Fixed

**Pull request merged**